src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -96,6 +97,16 @@ public Conversation getConversationWithTitle(String title) {
     return null;
   }
 
+  /** Find and return the Conversation with the given UUID. */
+  public Conversation getConversationWithId(UUID id) {
```

Comment:
Same comment as `getMessageWithId`

Reply:
This function is used in `ActivityFeedServlet` in order to find which conversation a message in sent in to include this detail in the `output` string of an `Activity`.